kornia/contrib/models/rt_detr/post_processor.py

```diff
@@ -38,8 +38,8 @@ def forward(self, logits: Tensor, boxes: Tensor, original_sizes: list[ImageSize]
         boxes_xy = concatenate([cxcy - wh * 0.5, wh], -1)
 
         sizes_wh = torch.empty(1, 1, 2, device=boxes.device, dtype=boxes.dtype)
-        sizes_wh[..., 0] = original_sizes[0].width
-        sizes_wh[..., 1] = original_sizes[0].height
+        sizes_wh[..., 0] = original_sizes[0][0]
+        sizes_wh[..., 1] = original_sizes[0][0]
```

Comment:
```suggestion
        sizes_wh[..., 1] = original_sizes[0][1]
```
you made your own bug here :)